### PR TITLE
serialize auth codes in rpc

### PR DESCRIPTION
**Pull request by majestrate**
serialize auth codes for endpoints in rpc so the gui can see them

### CODE REVIEW

**Comment by majestrate** [comment]
this needs to go in before we tag

File: llarp/router_contact.hpp [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 namespace lokimq
 {
-  struct bt_list_consumer;
+  class bt_list_consumer;
```
Comment (tewinget) [inline llarp/router_contact.hpp:22]: for some reason I conflated this with the list producer I was working on when you mentioned it, so I was confused.  ty for fixing.

**Review by tewinget — APPROVED** [review]
looks fine

**Comment by tewinget** [comment]
Linter is unhappy, should clang-format before merge.

**Comment by majestrate** [comment]
rebased

File: llarp/router_contact.hpp (hunk 1 above)
Comment (majestrate) [reply llarp/router_contact.hpp:22]: yw

**Pull request merged**